pyproject.toml

```diff
@@ -1,6 +1,7 @@
 [build-system]
 requires = ["setuptools", "wheel", "setuptools-rust"]
+build-backend = "setuptools.build_meta"
```

Comment:
For those reviewing this PR in the future: `build-backend` refers to https://github.com/pypa/setuptools/blob/main/setuptools/build_meta.py